### ISSUE TITLE
Thu May 16, 2024

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/czi-scientific-python-mgmt#71]
Notes from Isabela's audit.

Page audited: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/web-components.html

- There are two separate hierarchies of headings on the page (section navigation and the main article)
- Should the "on this page" navigation have headings
  - Argument against: the area can be reached via landmarks
  - Argument for: the section navigation has heading so it feels like a inconsistency in UX
- The light/dark toggle button does not communicate its state
- Sphinx-togglebutton expand/collapse button does not communicate its state
- Sphinx-copybutton is not visible until hovered or focused
- Tabs need other hover indicator other than just color change (underlining?)
- Dropdown admonition chevrons hover effect is a little too subtle?
- Copy button tool tip (native/browser does not meet criteria, does the other one meet the criteria, do we need both?)
- Not enough space between navbar icon buttons
- Not enough space between the links in the "on this page" navigation
